micro-airtable-api.test.js

```diff
@@ -0,0 +1,77 @@
+const http = require('http');
+const got = require('got');
+const listen = require('test-listen');
+const handler = require('./index');
+
+const describeIfEnv = (process.env.AIRTABLE_API_KEY && process.env.AIRTABLE_BASE_ID) ? describe : xdescribe;
+const baseConfig = {
+  airtableBaseId: process.env.AIRTABLE_BASE_ID,
+  airtableApiKey: process.env.AIRTABLE_API_KEY,
+};
+
+const getExistingPosts = async (client) => {
+  return (await client.get('/v0/Posts', {json: true})).body;
```

Comment:
Can we return `response.body.records` instead of just `response.body`?

Since the method name is `getExistingPosts`, I'd assume it would return a set of posts, and not an object that has a `records` property that has the posts. Might just encapsulate things a little better.